front/src/modules/errors/components/AppErrorBoundary.tsx

```diff
@@ -0,0 +1,20 @@
+import { ErrorInfo, ReactNode } from 'react';
```

Comment:
this one in error-handler folder (this is the feature folder vs /errors which is a new authorized subfolder like components)